test/testhdf5.h

```diff
@@ -161,94 +164,70 @@
         }                                                                                                    \
     } while (0)
 
-/* Used to document process through a test */
-#if defined(H5_HAVE_PARALLEL) && defined(H5_PARALLEL_TEST)
-#define MESSAGE(V, A)                                                                                        \
```

Comment:
Moved into testframe.h and simplified by using the new 'test process ID' parameter